templates/aliases.j2

```diff
@@ -1,4 +1,4 @@
 # Managed by Ansible (role: mailclient), do not edit!
 {% for item in mailclient_to_redirect %}
-{{ item }}: {{ mailclient_system_owner }}@{{ mailclient_domain }}
+{{ item }}: {{ mailclient_system_owner }}
```

Comment:
Can we somehow make this backwards-compatible? Quite a few deployments break if we do this.

Reply:
Uff. New variable? But that would make it really ugly...
I'm not sure if backwards compatibility is worth the hassle.

Reply:
Hmm - how about a flag `mailclient_system_owner_include_domain` which defaults to true?

Reply:
See 0fdb7da. I don't really like it, but that's me :stuck_out_tongue: